src/components/NewBeerForm.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Redirect } from 'react-router-dom';
+
+class NewBeerForm extends Component {
+  state = {
+    name: "",
+    tagline: "",
+    description: "",
+    first_brewed: "",
+    brewers_tips: "",
+    attenuation_level: "",
+    contributed_by: "",
+  }
+
+  handleNameChange = (event) => {
+    event.preventDefault();
+    const value = event.target.value;
+    this.setState({
+      name: value,
+    })
+  }
+
+  handleTaglineChange = (event) => {
+    event.preventDefault();
+    const value = event.target.value;
+    this.setState({
+      tagline: value,
+    })
+  }
```

Comment:
Ojo, todos los handlers los puedes sustituir por uno único:

``const { name, value } = event.target;
    this.setState({
      [name]: value
    });``